### PR TITLE
Refactored Index Page

### PR DESCRIPTION
**Pull request by Steve-Leke**
The patient and practitioner buttons on the index.html page weren't navigating to the right pages when clicked.

I changed the hrefs for the buttons, so when clicked they would navigate the user to the practitioner and patient sign up page respectively.

Brief Explanation:
- Updated the hrefs for patient and practitioner buttons on index.html.

### CODE REVIEW

File: templates/prSignUp.html [hunk 1]
```diff
@@ -1,33 +1,32 @@
 <!DOCTYPE html>
```
Comment (j0n3lson) [inline templates/prSignUp.html]: Are changes in this file relevant to the bug you found?

**Review by j0n3lson — CHANGES_REQUESTED** [review]

File: templates/prSignUp.html (hunk 1 above)
Comment (Steve-Leke) [reply templates/prSignUp.html]: No the changes on the right seem to be the original version of the code we edited. It's not relevant at all.

I don't know how it reverted the changes, (maybe they were stashed or something) but we don't want the code in. green.

**Comment by j0n3lson** [comment]
Probably just delete the prSignUp file from this change.

**Comment by Steve-Leke** [comment]
Okay, I deleted it!

**Review by j0n3lson — APPROVED** [review]

**Pull request merged**